base/i18n/icu_util.cc

```diff
@@ -156,6 +161,7 @@ void LazyInitIcuDataFile() {
   wcscpy_s(tmp_buffer, data_path.value().c_str());
   debug::Alias(tmp_buffer);
 #endif
+  data_path = data_path.AppendASCII("icu");
```

Comment:
Should be wrapped in macros

Reply:
Made a mistake to add this line here, it's actually not needed.